boards/arm/nucleo_f401re/nucleo_f401re.dts

```diff
@@ -54,6 +54,9 @@
 	status = "ok";
 };
 
+arduino_i2c: &i2c1 {};
```

Comment:
This requires doc update, but same change done in #12334. You can drop this commit

Reply:
ack